### DISCUSSION TITLE
Sponsoring a forum

### DISCUSSION THREAD

**Discussion by Justman100** [leokhoa/laragon#745] [Ideas]
Hi, had heard that the forum was shut down! Could provide a new forum 🙂...

**Comment by Justman100** [comment]
@leokhoa

**Comment by leokhoa** [comment]
Thanks @Justman100 
It would be nice if you can sponsor me via Github or Paypal: https://github.com/leokhoa/laragon/discussions/383

**Reply by Justman100** [reply]
Well, I was thinking that I could provide the forum directly

**Reply by Sophist-UK** [reply]
I have also offered to host a forum for nothing. No response.

**Reply by Justman100** [reply]
@leokhoa

**Reply by rizaldywirawan** [reply]
Isn't GitHub discussion is enough?

**Reply by Justman100** [reply]
> Isn't GitHub discussion is enough?

Would be more professional

**Reply by Sophist-UK** [reply]
On the list of things I would personally like to see, a Forum is low on the list (because as you say we have discussions).

My reason for offering to host was more to try to get @leokhoa to come out of his shell and engage more with his community of enthusiastic and supportive users.

**Reply by Justman100** [reply]
> On the list of things I would personally like to see, a Forum is low on the list (because as you say we have discussions).
> 
> My reason for offering to host was more to try to get @leokhoa to come out of his shell and engage more with his community of enthusiastic and supportive users.

Well, I would not only offer the forum, but also set it up, maintain and manage it...

**Reply by Sophist-UK** [reply]
Ditto - but @leokhoa doesn't seem interested.

**Reply by AWqxKAWERbXo** [reply]
> Would be more professional

I don't agree. GitHub is a code platform - also for professionals. And it's likely here to stay, so it's maintenance-free infrastructure.

**Reply by Justman100** [reply]
> > Would be more professional
> 
> I don't agree. GitHub is a code platform - also for professionals. And it's likely here to stay, so it's maintenance-free infrastructure.

Maintenance would be my responsibility

**Reply by Sophist-UK** [reply]
Exactly who first said "Would be more professional" that was quoted by @alxndr-w as if someone had actually said it, because I cannot find a comment that says this.

The "we don't need a web-site because Github is good enough" argument could be applied to each and every one of the thousands of most successful and most widely used projects on Github - but funnily enough most of them DO have web sites. Why would that be? Probably because it looks more professional and because it gets wider publicity and ...

**Reply by Justman100** [reply]
> Exactly who first said "Would be more professional" that was quoted by @alxndr-w as if someone had actually said it, because I cannot find a comment that says this.
> 
> The "we don't need a web-site because Github is good enough" argument could be applied to each and every one of the thousands of most successful and most widely used projects on Github - but funnily enough most of them DO have web sites. Why would that be? Probably because it looks more professional and because it gets wider publicity and ...

That's what I say

**Reply by AWqxKAWERbXo** [reply]
I guess my comment needs some explanation. 

First of all, yes, a support forum that is in active support could be great...

... until, which can be a risk, someone does not feel responsible anymore. Then the content might get lost (again), the forum software gets insecure.

That's one reason I'd prefer a hosted solution with shared responsibilities. 

I also think @Sophist-UK @Justman100 your engagement adds more value here in e.g. active support, documentation or moderating.

My next argument is GitHub as a platform with e.g. Copilot can add more value to the content we create here, like searching for help, writing tutorials, etc. A classic forum / board software feels "out of time" to me. 

So what others do/did in the past is not really an argument to me to decide how we could organise information and support in future as a community.

**Reply by AWqxKAWERbXo** [reply]
> if someone had actually said it, because I cannot find a comment that says this.

@Justman100 wrote that, as he confirmed, just a few comments before. :)

**Reply by Justman100** [reply]
> I guess my comment needs some explanation.
> 
> First of all, yes, a support forum that is in active support could be great...
> 
> ... until, which can be a risk, someone does not feel responsible anymore. Then the content might get lost (again), the forum software gets insecure.
> 
> That's one reason I'd prefer a hosted solution with shared responsibilities.
> 
> I also think @Sophist-UK @Justman100 your engagement adds more value here in e.g. active support, documentation or moderating.
> 
> My next argument is GitHub as a platform with e.g. Copilot can add more value to the content we create here, like searching for help, writing tutorials, etc. A classic forum / board software feels "out of time" to me.
> 
> So what others do/did in the past is not really an argument to me to decide how we could organise information and support in future as a community.

It's more about being professional! Anything a 10 year old kiddi can do is unprofessional, boring, not worthy, etc
